src/views/admin/donates.hbs

```diff
@@ -0,0 +1,34 @@
+<main class="main">
+  <section class="taps">
+    <input type="button" value="New Donations" class="new">
+    <input type="button" value="Current Donations" class="current">
+    <input type="button" value="Done Donations" class="done">
+  </section>
+  {{#each card}}
+  {{#unless card.state}}
+  <section class="cards--new">
+    <div class="card--taps">
+      <input type="button" value="Personal Info" class="tap--personal">
+      <input type="button" value="Donation" class="tap--donation">
+    </div>
+    {{!-- {{#if !card.state}} --}}
+    <div class="cards--info">
+      <div class="cards--info-personal">
+        <h4 class="fullName">Full Name: {{fullName}}</h4>
+        <h4 class="phoneNumber">Phone Number: {{phoneNumber}}</h4>
+        <h4 class="email">Email: {{email}}</h4>
+        <h4 class="city">City: {{cityName}}</h4>
```

Comment:
@AbdallahAmmar96 Are fullName, phoneNumber, email and cityName attributes of the current card in the `#each` loop?  I think then you'd need to use `this.fullName`, `this.phoneNumber`, and so on.
Or it looks like handlebars 3.0 has a feature called block parameters which makes it easier to write `#each` loops, if you're interested:  https://handlebarsjs.com/block_helpers.html#block-params